.github/workflows/conformance-externalworkloads-v1.10.yml

```
@@ -72,8 +72,8 @@ jobs:
     name: Deduce required tests from code changes
     if: |
       (github.event_name == 'issue_comment' && (
-        (startsWith(github.event.comment.body, 'ci-external-workloads-v1.10') ||
-        startsWith(github.event.comment.body, 'test-backport-1.10')) &&
+        startsWith(github.event.comment.body, 'ci-external-workloads-v1.10') ||
+        startsWith(github.event.comment.body, 'test-backport-1.10')
```

This should be:
```suggestion
        startsWith(github.event.comment.body, 'ci-external-workloads-v1.10') ||
        (startsWith(github.event.comment.body, 'test-backport-1.10'))
```
I am joining a patch with this change and another one.

---

Why? Does it make a difference?

---

No, it's just to be consistent with the other workflows (just like for the other comment above). The failure was indeed related to the `&&`, but GH suggestion shows the diff w.r.t. to the new code rather than the original one.